docs/usage/installation.md

```diff
@@ -77,4 +108,86 @@ rd-agent/
 
 :::
 
+## Verification <Badge type="warning" text="Since 5.3" />
+
+To ensure trust and security in the software supply chain, verification of release artifacts
+confirms that the downloaded files are authentic, unaltered, and originate from the intended source.
+This process helps protect users from tampered or malicious builds by using cryptographic proofs of
+provenance. Each agent release includes metadata that allows users to independently verify the
+origin and integrity of both the archive files and container images.
+
+### Release Archives
+
+Each agent archive is accompanied by a provenance file in [SLSA](https://slsa.dev) format, which
+verifies the authenticity and integrity of the downloaded files. The `slsa-verifier` tool checks
+that the artifact was built from the correct source and tag. <sup>[1](#slsa-verifier)</sup>
+
+::: code-group
+
+```shell-vue [CD Agent]
+wget {{ download }}/cd-agent.tar.gz.intoto.jsonl
+slsa-verifier verify-artifact cd-agent.tar.gz \
+  --source-uri github.com/medizininformatik-initiative/fts-next \
+  --source-tag {{ release }} \
+  --provenance-path cd-agent.tar.gz.intoto.jsonl
+```
+
+```shell-vue [TC Agent]
+wget {{ download }}/tc-agent.tar.gz.intoto.jsonl
+slsa-verifier verify-artifact tc-agent.tar.gz \
+  --source-uri github.com/medizininformatik-initiative/fts-next \
+  --source-tag {{ release }} \
+  --provenance-path tc-agent.tar.gz.intoto.jsonl
+```
+
+```shell-vue [RD Agent]
+wget {{ download }}/rd-agent.tar.gz.intoto.jsonl
+slsa-verifier verify-artifact rd-agent.tar.gz \
+  --source-uri github.com/medizininformatik-initiative/fts-next \
+  --source-tag {{ release }} \
+  --provenance-path rd-agent.tar.gz.intoto.jsonl
+```
+
+:::
+
+If the verification passes, the output should contain confirmation that the artifact is valid
+and matches the signed provenance:
+
+```
+PASSED: SLSA verification passed
+```
+
+---
+
+<small id="slsa-verifier">
+  1. Please see official <code>slsa-verifier</code> <a href="https://github.com/slsa-framework/slsa-verifier#installation">installation</a> instructions
+</small>
+
+### Container Images
+
+The release archives come with a `verify.sh` utility script that allows verifying the provenance of
```

Comment:
Maybe:
Each release archive includes a verify.sh utility script for verifying the provenance of the container images used in the corresponding Compose files.

Reply:
Maybe we should give a short example on how to use the verify script.